src/components/ArticlePreview.jsx

```diff
@@ -48,13 +36,10 @@ export default class ArticlePreview extends React.Component {
 ArticlePreview.propTypes = {
   article: React.PropTypes.shape({
     title: React.PropTypes.string.isRequired,
-    image: React.PropTypes.string.isRequired,
-    slug: React.PropTypes.string.isRequired,
+    teaser: React.PropTypes.string.isRequired,
```

Comment:
so did we decide to standardize this as teaser everywhere? I've just been seeing a lot of diff names so I'm wondering if we are settled on this one.